### ISSUE TITLE
Add Tests Controller for CH31

### ISSUE THREAD

**Issue by cohnjesse** [department-of-veterans-affairs/va.gov-team#12090]
# Background
The Chapter 31 form will be submitting to central mail. Submitting a form to central mail within VA.gov means we need to set up a controller on the back end to receive the payload from the front end and call the CentralMail service. In order to submit a PR to `vets-api` we also need our controller to be unit tested to 90% coverage so this ticket includes that work as well.

## Consideration
- Test for the controller can be written once Confirmation from FE that they can accommodate the payload change request
- [MODEL] Pending the adjustment of the payload from FE, test may resume (separate ticket)

## Tasks
- [x] Set up controller file
- [x] Define methods in the controller to handle the payload from the frontend
- [x] Create unit tests for this controller file up to 90% coverage
- [x] Add Swagger docs, if necessary
- [x] Submit a PR to `vets-api`

## Acceptance criteria
- [x] We have an API endpoint set up to accept payload from the front end and submit to central mail
- [x] A PR has been submitted for review

**Comment by digitaldrk** [comment]
This PR is held up by the model PR: https://github.com/department-of-veterans-affairs/vets-api/pull/5321

Model PR: https://github.com/department-of-veterans-affairs/vets-api/pull/5320

Large Host PR: https://github.com/department-of-veterans-affairs/vets-api/pull/5209
